mod_reforged/hooks/skills/perks/perk_nimble.nut

```diff
@@ -91,4 +98,14 @@
 			_properties.OffensiveReachIgnore += 1;
 		}
 	}
+
+	q.getStaminaModifierThreshold <- function()
+	{
+		local b = this.getContainer().getActor().getBaseProperties().getClone();
+		foreach (trait in this.getContainer().getSkillsByFunction(@(s) s.isType(::Const.SkillType.Trait) || s.isType(::Const.SkillType.PermanentInjury)))
+		{
+			trait.onUpdate(b);
+		}
+		return ::Math.max(0, 0.15 * b.Stamina * (b.StaminaMult >= 0 ? b.StaminaMult : 1.0 / b.StaminaMult));
```

Comment:
the `0.15` should be a member variable for easier changing by future balancing commits or submods.